pypeit/spectrographs/keck_lris.py

```diff
@@ -51,8 +53,9 @@ def default_pypeit_par():
         par['calibrations']['biasframe']['exprng'] = [None, 1]
         par['calibrations']['darkframe']['exprng'] = [999999, None]     # No dark frames
         par['calibrations']['pinholeframe']['exprng'] = [999999, None]  # No pinhole frames
-        par['calibrations']['pixelflatframe']['exprng'] = [None, 30]    # This may be too low for LRISb
-        par['calibrations']['traceframe']['exprng'] = [None, 30]
+        par['calibrations']['pixelflatframe']['exprng'] = [None, 60]
+        par['calibrations']['traceframe']['exprng'] = [None, 60]
+        par['calibrations']['standardframe']['exprng'] = [None, 30]
 
```

Comment:
see: https://www2.keck.hawaii.edu/inst/lris/spectroscopic_flats.html Many users will use this as guidance on the exp time.  On blue side most above 60.  If I understand this, it will look for <= 60, so may need to up further?

Reply:
Good suggestion.  Have made this change for the blue side.